lua/lib/filters.lua

```diff
@@ -81,18 +81,18 @@ setmetatable(median, { __index=f })
 --- @param bufsize: window size, cannot change after creation
 function median.new(bufsize)
    local new = setmetatable({}, median)
-   
+
    new.buf = {}
    if bufsize==nil then bufsize=17 end
    -- only odd buffer sizes are allowed!
    if (bufsize) == 0 then bufsize = bufsize + 1 end
```

Comment:
nothing to do with your PR, but this looks like a typo where bufsize is supposed to be coerced to (2*N)+1, but the modulo comparison is missing